dcpy/test/conftest.py

```diff
@@ -97,6 +98,12 @@ def _test_product(
     yield constants
 
 
+@pytest.fixture(scope="function")
+def tmp_dir():
```

Comment:
can we just use [tmp_path](https://docs.pytest.org/en/stable/how-to/tmp_path.html)?